### PR TITLE
Added sidebar to the viewer

### PR DESCRIPTION
**Pull request by tacruc**
Signed-off-by: Arne Hamann <kontakt+github@arne.email>

### CODE REVIEW

File: templates/index.php [hunk 1]
```diff
@@ -21,6 +21,7 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>
  *
  */
+script('maps', 'mockupFilesApp');
```
Comment (eneiluj) [inline templates/index.php:24]: What is this? It still works if I remove it.

File: src/contactsController.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { generateUrl } from '@nextcloud/router';
 
 import { basename, formatAddress } from './utils';
+import escapeHTML from 'escape-html';
```
Comment (eneiluj) [inline src/contactsController.js:4]: Could you remove those which are out of topic?

File: src/photosController.js [hunk 3]
```diff
@@ -205,7 +207,7 @@ PhotosController.prototype = {
                 iconUrl = _app.getImageIconUrl();
             }
             var label = cluster.getChildCount();
-            if( availZoomLevels == 0 && label > 1){ 
+            if( availZoomLevels == 0 && label > 1){
```
Comment (eneiluj) [inline src/photosController.js:210]: Could you also remove those changes on trailing white spaces? Or maybe fix the code style in this file?

This line could be:
``` javascript
if (availZoomLevels === 0 && label > 1) {
```

File: templates/index.php (hunk 1 above)
Comment (tacruc) [reply templates/index.php:24]: I thougt I allready removed it.

File: src/contactsController.js (hunk 2 above)
Comment (tacruc) [reply src/contactsController.js:4]: My problem is that without this my dev setup is not working, as it is running NC20. But I can bring it up in a own PR.

**Comment by tacruc** [comment]
> Uncaught TypeError: can't access property "extend", OCA.Files.DetailTabView is undefined
> Uncaught TypeError: can't access property "Client", t.Files is undefined
> Uncaught TypeError: can't access property "extend", OCA.Files.DetailTabView is undefined

I thing the errors are a problem we are not going to solve now. They happen for the photos app, too. I think it is related to different tabs which are not yet migrated.

**Comment by eneiluj** [comment]
Ok so ready to merge after having removed the `mockupFilesApp` script. Tell me if you don't have time to do it, I'll do it.

**Comment by skjnldsv** [comment]
> I thing the errors are a problem we are not going to solve now. They happen for the photos app, too. I think it is related to different tabs which are not yet migrated.

Yep, you can ignore those :)

**Pull request merged**